tests/components/datadog/test_init.py

```diff
@@ -160,17 +149,17 @@ def test_state_changed(self, mock_connection, mock_client):
                     ]
                 )
 
-            assert mock_client.gauge.call_args == mock.call(
+            assert mock_statsd.gauge.call_args == mock.call(
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6356951)